index.html

```diff
@@ -8,34 +8,204 @@
+                </div> 
+
+            </div>
+        </section>
+
+        <section>
+            <div class="border_project4">
+                <div class="image_tonic">
+                    <img src="image/Snapshoot Portfolio.png" alt="Snapshoot Portfolio">
+                </div>
+        
+                <h1 class="tonic">Tonic</h1>
+                <div class="grid-container">
+                    <div class="grid-item1">CANOPY</div>
+                    <div class="grid-item2"><img src="image/Counter.svg" alt="Counter"></div>
+                    <div class="grid-item3">Back End Dev</div>
+                    <div class="grid-item4"><img src="image/Counter.svg" alt="Counter"></div>
+                    <div class="grid-item5">2015</div>
+                </div>
+                <article>
+                    <p class="canopy">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                </article>
+                <div class="languages">
+                    <div class="lang1"><img src="image/Tag_html.svg" alt="HTML Tag"></div>
+                    <div class="lang2"><img src="image/Tag_css.svg" alt="HTML Tag"></div>
+                    <div class="lang3"><img src="image/Tag.svg" alt="HTML Tag"></div>
+                </div> 
+                <div class="nav-button">
+                    <a href="https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m2_mobile/works-mobile.png"><img src="image/Enabled.svg" alt="See Project button"></a> 
+                </div>
+            </div>
+        </section>
+
+        <section>
+            <div class="border_project5">
+                <div class="image_project5">
+                    <img src="image/art_printing.svg" alt="Professional Art Printing">
+                </div>
+                <header class="stories"><img src="image/Project Title.svg" alt="Project Title">
+                </header>
+                <div class="grid-container">
+                    <div class="grid-item1">CANOPY</div>
+                    <div class="grid-item2"><img src="image/Counter.svg" alt="Counter"></div>
+                    <div class="grid-item3">Back End Dev</div>
+                    <div class="grid-item4"><img src="image/Counter.svg" alt="Counter"></div>
+                    <div class="grid-item5">2015</div>
+                </div>
+                <article>
+                    <p class="canopy">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                </article>
+                <div class="languages">
+                    <div class="lang1"><img src="image/Tag_html.svg" alt="HTML Tag"></div>
+                    <div class="lang2"><img src="image/Tag_css.svg" alt="HTML Tag"></div>
+                    <div class="lang3"><img src="image/Tag.svg" alt="HTML Tag"></div>
+                </div> 
```

Comment:
- [ ] Here also, it's better to use the unordered list `ul and li` instead of `div.` for semantic reasons and user experience (UX).